misk-launchdarkly-core/src/main/kotlin/misk/feature/launchdarkly/LaunchDarklyFeatureFlags.kt

```diff
@@ -132,7 +149,7 @@ class LaunchDarklyFeatureFlags @Inject constructor(
       }
     }
     if (attributes.number != null) {
-      attributes.number!!.forEach { (k, v) -> builder.privateCustom(k, v) }
+      attributes.number!!.forEach { (k, v) -> builder.privateCustom(k, v.toInt()) }
```

Comment:
Uncertain about what to do here, there is no longer a generic `number` option. The [only choices](http://launchdarkly.github.io/java-server-sdk/com/launchdarkly/sdk/LDUser.Builder.html#privateCustom-java.lang.String-int-) are `privateCustom(k : String!, n: Int)` or `privateCustom(k : String!, n: Double)`.

Reply:
we may need to make an api change here, but we could probably figure out the passed in type and do it that way.